src/pages/home/report/ParticipantLocalTime.js

```diff
@@ -56,7 +56,7 @@ class ParticipantLocalTime extends PureComponent {
     render() {
         const reportRecipientDisplayName = this.props.participant.firstName
             || (Str.isSMSLogin(this.props.participant.login)
-                ? this.props.toLocalPhone(this.props.participant.displayName)
+                ? this.props.formatPhoneNumber(Str.removeSMSDomain(this.props.participant.displayName))
```

Comment:
You mean to create a new util that will check if the login isSMSlogin and if so format the phone number or return the login unformatter otherwise? Or you mean just removing SMS domain to format the phoneNumber? Regarding the first option - as we will be changing the logic for displaying the name (if user has a phoneNumber as a login but has also provided real name) I wouldn't do that now. If you meant the latter one - I will move the Str. removeSMSDomain to the formatPhoneNumber function - it won't affect the phone numbers without the sms domain so it can be moved there.

Reply:
Ok, let's do the latter for now. 

And we can do the former in the follow up issue.